msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -37,6 +37,550 @@ The following table provides a list of APIs available across the Microsoft Teams
 
 To identify and retrieve contextual information for your tab content, see [get context for your Teams tab](../tabs/how-to/access-teams-context.md#get-context-by-using-the-microsoft-teams-javascript-library). `meetingId` is used by a tab running in the meeting context and is added for the response payload.
 
+### Examples
+
+The following are the TeamsJS v2 responses for Get user context API based on meeting type, user type, and call type:
+
+* Meeting type
+
+   # [Channel meeting](#tab/channel-meeting)
+
+   The following is a JSON payload response for a channel meeting for in-tenant users:
+
+   ```json
+    {
+     "app": {
+      "locale": "en-us",
+      "sessionId": "ff47ec00-e6a7-4dc1-a6ae-f44110f50c94",
+      "theme": "default",
+      "iconPositionVertical": 0,
+      "osLocaleInfo": {
+       "platform": "windows",
+       "regionalFormat": "en-in",
+       "shortDate": "dd-MM-yyyy",
+       "longDate": "dd MMMM yyyy",
+       "shortTime": "HH:mm",
+       "longTime": "HH:mm:ss"
+      },
+      "parentMessageId": "1678109354022",
+      "userClickTime": 1678109521159,
+      "userFileOpenPreference": "inline",
+      "host": {
+       "name": "Teams",
+       "clientType": "desktop",
+       "sessionId": "c3c3c0a0-f7a1-b070-6b89-c8cd1f380042",
+       "ringId": "ring1"
+      },
+      "appLaunchId": "7346ae66-5cac-47f9-8a0d-1228dac474cb"
+     },
+     "page": {
+      "id": "Test",
+      "frameContext": "sidePanel",
+      "subPageId": "",
+      "isFullScreen": false,
+      "isMultiWindow": true,
+      "sourceOrigin": ""
+     },
+     "user": {
+      "id": "57efa5f3-273c-47e2-a871-4879e5d849cf",
+      "displayName": "",
+      "isCallingAllowed": undefined,
+      "isPSTNCallingAllowed": undefined,
+      "licenseType": "Unknown",
+      "loginHint": "v-prkamble@microsoft.com",
```

Comment:
Expanding on the comments below it might be better to change these to a test user from one of the sample tenants.